topdown/object.go

```diff
@@ -26,6 +26,25 @@ func builtinObjectUnion(_ BuiltinContext, operands []*ast.Term, iter func(*ast.T
 	return iter(ast.NewTerm(r))
 }
 
+func builtinObjectUnionN(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+	arr, err := builtins.ArrayOperand(operands[0].Value, 1)
```

Comment:
We could allow sets of objects, too. What do you think?

Reply:
https://github.com/open-policy-agent/opa/issues/4012#issuecomment-970077620

😉

Reply:
😂 ignore me.